_data/simple-icons.json

```diff
@@ -7743,7 +7747,7 @@
         {
             "title": "Robot Framework",
             "hex": "000000",
-            "source": "https://github.com/robotframework/visual-identity",
+            "source": "https://github.com/robotframework/visual-identity/blob/master/logo/robot-framework.svg",
             "guidelines": "https://github.com/robotframework/visual-identity/blob/master/robot-framework-brand-guidelines.pdf"
```

Comment:
Should be https://github.com/robotframework/visual-identity/blob/fadf8cda9f79ea31987a214f0047cca9626327b7/robot-framework-brand-guidelines.pdf